lib/Dialect/FIRRTL/Transforms/LowerLayers.cpp

```diff
@@ -748,27 +806,43 @@ void LowerLayersPass::runOnOperation() {
       includes.append("\n");
     }
 
+    hw::OutputFileAttr bindFile;
+    if (auto outputFile =
+            layerOp->getAttrOfType<hw::OutputFileAttr>("output_file")) {
+      auto dir = outputFile.getDirectoryAttr().getValue();
+      bindFile = hw::OutputFileAttr::getFromDirectoryAndFilename(
+          &getContext(), dir, prefix + ".sv",
+          /*excludeFromFileList=*/true);
+    } else {
+      bindFile = hw::OutputFileAttr::getFromFilename(
+          &getContext(), prefix + ".sv", /*excludeFromFileList=*/true);
+    }
```

Comment:
Having an empty `auto dir` might make this cleaner as both code paths can use `getFromDirectoryAndFilename`.

Reply:
If dir is empty, I believe getFromDirectoryAndFilename will create a path "/<filename>", (a file in the root), rather than a relative directory name, which is what we want in this case.